src/classes/AccountDataFactory.cls

```diff
@@ -1,4 +1,4 @@
-@IsTest
+//@IsTest
 public class AccountDataFactory extends DataFactory {
```

Comment:
Classes must use `with sharing` to enforce record level access and avoid bypassing record level security